Content.Shared/DeltaV/FeedbackOverwatch/SharedFeedbackOverwatchSystem.Events.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Content.Shared.DeltaV.FeedbackOverwatch;
+
+public sealed partial class SharedFeedbackOverwatchSystem
+{
+    private void InitializeEvents()
+    {
+        // Subscribe to events that would be good for popups here. If possible, do this in the new systems themselves
+        // to reduce merge conflicts!
```

Comment:
clarify that this is just for avoiding changes to upstream code and that any deltav specific features should just use their own system